docs/web/errors.mdx

```diff
@@ -170,6 +197,33 @@ try {
 }
 ```
 
+## Timeouts
```

Comment:
We have had a couple of questions from users on how to cancel a call, which means that our documentation for cancellation is not discoverable enough at the bottom of the "Errors" page.

Let's try adding a new page "Timeouts & cancellation" below "Headers & trailers" instead.